kubernetes/resource_kubernetes_deployment.go

```diff
@@ -408,15 +408,20 @@ func waitForDeploymentReplicasFunc(conn *kubernetes.Clientset, ns, name string)
 			return resource.NonRetryableError(err)
 		}
 
+		var specReplicas int32 = 1 // default, acording to API docs
```

Comment:
According to Go docs, int32 is the default size of `int` on a 32-bit system. But on 64-bit systems, it's the size of int64. Why specify the size?

Reply:
It's the type of `*dply.Spec.Replicas`. Figured I should go with the same.